internal/utils/utils.go

```diff
@@ -9,21 +9,13 @@ package utils
 
 import (
 	"crypto/sha1"
-	"fmt"
 	"math/rand"
-	"os"
-	"runtime"
 	"time"
 )
 
-// GenerateMessageId отдает по сути unix timestamp но ужасно специфическим образом
-// TODO: нахуя нужно битовое и на -4??
```

Comment:
btw have you got the idea? 😄

Reply:
> btw have you got the idea? smile

I really don't have an idea. It seems like code from indian stackoverflow ))